test/integration/model/json.test.js

```diff
@@ -227,7 +227,7 @@ describe(Support.getTestDialectTeaser('Model'), function () {
           return this.Event.findAll({
             where: {
               data: {
-                employment: null
+                employment: current.dialect.name === 'mysql' ? 'null' : null
```

Comment:
This seems wrong to me - We are inserting literal null, but querying for the string null. Is this because it is actually inserted as the JSON value null, and not as a database null? What does postgres do in this case?